lib/mule.js

```diff
@@ -309,13 +307,22 @@ var STATTAGS = 'MaxHitPoints MaxMagicPoints Attack Defense Speed Dexterity HpReg
 var STATABBR = 'HP MP ATT DEF SPD DEX VIT WIS'.split(' ')
 Mule.prototype.parse = function(data) {
 	if (this.overlay) this.overlay.hide()
-
-	var d = data.query.results.Chars
+    console.log(data);
+    if(data.Chars){
+	    var d = data.Chars
+    } else {
+        var d = data.query.results.Chars
+    }
 	d = {
 		Char: d.Char,
 		Account: d.Account || {}
 	}
-	data.query.results.Chars = d
+    if(data.Chars){
+	    data.Chars = d
+    } else {
+        data.query.results.Chars = d
+    }
+	
```

Comment:
These switches are presumably for compatibility with yql? If you intend to have it as a fallback, actually do it.